messages.rb

```diff
@@ -276,6 +289,92 @@ def send_list(event)
   event.attach_file(File::open(tmpfile))
 end
 
+def send_community(event)
+  msg = event.content
+  tabs = parse_tabs(msg)
+  tabs.empty? ? ftabs = "" : ftabs = format_tabs(tabs)
+  tabs = (tabs.empty? ? [:SI, :S, :SL, :SU, :SS, :SS2] : tabs)
+
+  levels = Score.where(rank: 0, highscoreable_type: "Level").includes(:level).where(levels: {tab: tabs})
+  episodes = Score.where(rank: 0, highscoreable_type: "Episode").includes(:episode).where(episodes: {tab: tabs})
+  lvls = levels.count
+  eps = episodes.count
+  tls = levels.map{|s| s.score}.sum
+  tes = episodes.map{|s| s.score}.sum
+  if !(tabs&[:SS, :SS2]).empty? # in order to compare lvls and eps we have to exclude secrets
```

Comment:
Cute. I didn't know you could use `&` on arrays like this, I like it.

Reply:
Learned it just for this, apart from intersection, one can get the union with `|` and the difference with `-`.

Reply:
I've used `|` before, but never extended the concept in my brain to the logical conclusion. Nice.